CHANGELOG.md

```diff
@@ -21,6 +21,29 @@ Increment the:
   [#1733](https://github.com/open-telemetry/opentelemetry-cpp/pull/1733)
 * [BUILD] Add CMake OTELCPP_PROTO_PATH [#1730](https://github.com/open-telemetry/opentelemetry-cpp/pull/1730)
 
+Deprecation notes:
+
+* [Deprecation] Deprecate experimental semantic conventions
+  [#1744](https://github.com/open-telemetry/opentelemetry-cpp/pull/1744)
+  * The file
+    `api/include/opentelemetry/trace/experimental_semantic_conventions.h`
+    is deprecated, and will be removed in a future release.
+    Use file
+    `api/include/opentelemetry/trace/semantic_conventions.h`
+    instead.
+
+  * The file
+    `sdk/include/opentelemetry/sdk/resource/experimental_semantic_conventions.h`
+    is deprecated, and will be removed in a future release.
+    Use file
+    `sdk/include/opentelemetry/sdk/resource/semantic_conventions.h`
+    instead.
+
+  * The function
+    `uint32_t hashCode(const char *str, uint32_t h = 0)`
```

Comment:
Also mention this is the function in global namespace which will be deprecated? As there is another one under `util::` namespace in ETW exporter.

Reply:
Thanks @ThomsonTan 

Fixed.